Tests/FunctionalTests/fss-simplified-search-results.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { chromium, Browser, BrowserContext, Page } from 'playwright'
+const { autoTestConfig } = require('./appSetting');
+const { pageObjectsConfig,pageTimeOut } = require('./pageObjects');
+import {LoginPortal,DataCollectionComparison,InsertSearchText} from './helpermethod'
+import{searchBatchAttribute,searchMultipleBatchAttributes,searchNonExistBatchAttribute} from './helperconstant'
+
+
+describe('Test Search Result Scenario On Simplified Search Page', () => {
+  jest.setTimeout(pageTimeOut.timeOutInMilliSeconds);
+  let browser: Browser;
+  let context: BrowserContext;
+  let page: Page;  
+
+  beforeAll(async () => {
+    browser = await chromium.launch({slowMo:100});
+    context = await browser.newContext();
+    page = await context.newPage();    
+    await page.goto(autoTestConfig.url)
+    await page.waitForTimeout(pageTimeOut.delay)
+    if((await page.$$(pageObjectsConfig.acceptCookieSelector)).length > 0){
+      await page.click(pageObjectsConfig.acceptCookieSelector);
+    }
+    page.click(pageObjectsConfig.searchButtonSelector);
+    await LoginPortal(page,autoTestConfig.user, autoTestConfig.password);    
+    
+    await page.waitForSelector(pageObjectsConfig.searchPageContainerHeaderSelector);
+    expect(await page.innerHTML(pageObjectsConfig.searchPageContainerHeaderSelector)).toEqual(pageObjectsConfig.searchPageContainerHeaderText);
+  
+    await page.click(pageObjectsConfig.simplifiedSearchLinkSelector);
+})
+
+  afterAll(async () => {
```

Comment:
Split the page and context close into an `afterEach` method

Reply:
Updated the scripts. Thanks.